setup.py

```diff
@@ -41,7 +27,7 @@ def get_version(*file_paths):
     ],
     description=("Command line tool, that converts Common Cartridge " "courses to Open edX Studio imports."),
     entry_points={"console_scripts": ["cc2olx=cc2olx.main:main"]},
-    install_requires=["lxml"],
+    install_requires=load_requirements("requirements/base.txt"),
```

Comment:
Shouldn't this be reading the `base.in` file and not the pinned `base.txt` file?

Reply:
I think, using `base.in` can cause unpredictable behaviour because not tested package versions can be used that can become a reason of conflicts. If we populate `constraints.txt`, it will be completely ignored and undesirable package versions will be installed. Am I wrong?